src/documentation/templates/models/NeutralColorsModel.json

```diff
@@ -5,198 +5,184 @@
-          "ratio": "4:8:1"
-        },
-        {
-          "class": "ms-bgColor-themeLighterAlt",
-          "ratio": "5:27:1"
-        },
-        {
-          "class": "ms-bgColor-white",
-          "ratio": "5:7:1"
-        }
-      ]
-    },
-    {
-      "name": "Neutral Secondary Alt",
-      "hex": "#767676",
-      "fontClass": "ms-fontColor-neutralSecondaryAlt",
-      "bgClass": "ms-bgColor-neutralSecondaryAlt",
-      "borderClass": "ms-borderColor-neutralSecondaryAlt",
-      "contrastBgs": [
-        {
-          "class": "ms-bgColor-white",
-          "ratio": "4:5:1"
-        }
-      ]
-    },
-    {
-      "name": "Neutral Tertiary",
-      "hex": "#A6A6A6",
-      "customNameColor": "ms-fontColor-black",
-      "fontClass": "ms-fontColor-neutralTertiary",
-      "bgClass": "ms-bgColor-neutralTertiary",
-      "borderClass": "ms-borderColor-neutralTertiary"
-    },
-    {
-      "name": "Neutral Tertiary Alt",
-      "customNameColor": "ms-fontColor-black",
-      "hex": "#C8C8C8",
-      "fontClass": "ms-fontColor-neutralTertiaryAlt",
-      "bgClass": "ms-bgColor-neutralTertiaryAlt",
-      "borderClass": "ms-borderColor-neutralTertiaryAlt"
-    },
-    {
-      "name": "Neutral Quaternary",
-      "hex": "#D0D0D0",
-      "customNameColor": "ms-fontColor-black",
-      "fontClass": "ms-fontColor-neutralQuaternary",
-      "bgClass": "ms-bgColor-neutralQuaternary",
-      "borderClass": "ms-borderColor-neutralQuaternary"
-    },
-    {
-      "name": "Neutral Quaternary Alt",
-      "customNameColor": "ms-fontColor-black",
-      "hex": "#DADADA",
-      "fontClass": "ms-fontColor-neutralQuaternaryAlt",
-      "bgClass": "ms-bgColor-neutralQuaternaryAlt",
-      "borderClass": "ms-borderColor-neutralQuaternaryAlt"
-    },
-    {
-      "name": "Neutral Light",
-      "hex": "#EAEAEA",
-      "customNameColor": "ms-fontColor-black",
-      "fontClass": "ms-fontColor-neutralLight",
-      "bgClass": "ms-bgColor-neutralLight",
-      "borderClass": "ms-borderColor-neutralLight",
-      "secondBgClass": "ms-bgColor-neutralPrimaryAlt"
-    },
-    {
-      "name": "Neutral Lighter",
-      "hex": "#004578",
-      "customNameColor": "ms-fontColor-black",
-      "fontClass": "ms-fontColor-neutralLighter",
-      "bgClass": "ms-bgColor-neutralLighter",
-      "borderClass": "ms-borderColor-neutralLighter",
-      "secondBgClass": "ms-bgColor-neutralPrimaryAlt"
-    },
-    {
-      "name": "Neutral Lighter Alt",
-      "hex": "#F8F8F8",
-      "customNameColor": "ms-fontColor-black",
-      "fontClass": "ms-fontColor-neutralLighterAlt",
-      "bgClass": "ms-bgColor-neutralLighterAlt",
-      "borderClass": "ms-borderColor-neutralLighterAlt",
-      "secondBgClass": "ms-bgColor-neutralPrimary"
-    },
-    {
-      "name": "White",
-      "hex": "#FFFFFF",
-      "customNameColor": "ms-fontColor-black",
+      "borderClass": "ms-borderColor-black"
+    },
+    {
+      "name": "Gray220",
```

Comment:
Just thinking out loud--should there be a `legacyName` value or something like that capturing the old name these map to?

Reply:
This file is only used for the internal documentation site, which I figured I should update but I don't think anyone is looking at. I'll add an `MDL2equivalent` property to each color that has one, and then we'll have the option to include this in the documentation if we want to.

Reply:
👍